### PR TITLE
Added support for measured units.

### PR DESCRIPTION
**Pull request by vivyzer**
1. Create usage based addons with measured units.
2. Create and list measured units.

### CODE REVIEW

**Comment by bhelx** [comment]
Thanks @vivyzer!

Code looks good at first glance. I will take a deeper look at it tonight or tomorrow morning.

File: src/main/java/com/ning/billing/recurly/model/MeasuredUnit.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        MeasuredUnit measuredUnit = (MeasuredUnit) o;
+
+        if (name != null ? !name.equals(measuredUnit.name) : measuredUnit.name != null) return false;
```
Comment (pierre) [inline src/main/java/com/ning/billing/recurly/model/MeasuredUnit.java]: Missing `id` comparison?

**Comment by vivyzer** [comment]
Added the id check..

**Comment by bhelx** [comment]
Thanks again @vivyzer 
Nice work!

**Pull request merged**